Tests/SwiftBoxMetricsTests/HandlersTests.swift

```diff
@@ -49,16 +15,44 @@ class StatsDHandlerTests: XCTestCase {
         }
     }
 
-    func testHandlerShouldGatherMetricsProperly() throws {
+    func testHandlerShouldGatherTimerMetricsProperly() throws {
         let handler = try StatsDMetricsHandler(
                 baseMetricPath: "test.path",
                 client: FakeStatsDClient()
         )
-        handler.sendMetric(metric: TimerMetric(name: "test", value: 1.01))
+        let timer = handler.makeTimer(label: "sample.timer", dimensions: [])
+        timer.recordNanoseconds(1_010_000)
+
+        let fakeClient = (handler.client as! FakeStatsDClient)
```

Comment:
why downcast?

Reply:
To access `gatheredMetrics`later in test.